### PR TITLE
Fixed blog featured post images that was cutoff

### LINKED ISSUE #292: Bug: featured post cuts off img
### Describe the bug

 The featured post is a different size from the rest of the posts. We _do not_ want to create multiple assets for every post. 

image

### Steps to reproduce

1. go to opensauced.pizza/blog
2. see the img on the feature post cut off

We should just standardize this. The featured post img can be the same sized.

### PR DESCRIPTION
**Pull request by dev-phantom**
## Description
This PR fixes the feature blog post images that was cutoff

## Related Tickets & Documents

Fixes #292 
## Mobile & Desktop Screenshots/Recordings
before:
image

after:
image

 ## Note 
 This pr also fixes the  minimum height issue on non featured post. this is to help maintain layout consistency
 
 before:
 
image

now:
image

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [ ] Tier 4

### CODE REVIEW

**Comment by dev-phantom** [comment]
@BekahHW 
please ma i need your review on this 

is it inline with the solution you suggested?

**Comment by BekahHW** [comment]
@dev-phantom this is looking much better. I did notice that the font seems to be stretched in the new version. Generally, we don't have titles in the image, so maybe we can just add a new image with the right dimensions.

## Updated

image


## Current

image

File: components/sections/blog/FeaturedPost.tsx [hunk 1]
```diff
@@ -17,9 +17,9 @@ const FeaturedPost: FC<FeaturedPostProps> = ({ data }): ReactElement => {
       <div className="w-full largeTablet:grid largeTablet:grid-cols-2 largeTablet:gap-x-11">
         <div className="hidden largeTablet:flex">
           <GradientBorderWrapper style={{ width: '100%' }}>
-            <div className="flex-1  h-[496px] relative rounded-[5px] overflow-hidden ">
-              <Image src={src as string} layout="fill" objectFit="cover" alt="Post cover" />
-            </div>
+          <div className="w-full relative rounded-[5px] overflow-hidden ">
+            <Image width="580" height={304} src={src as string} alt="Cover" />
```
Comment (nickytonline) [inline components/sections/blog/FeaturedPost.tsx:21]:
```suggestion
            <Image width={580} height={304} src={src as string} alt="Cover" />
```

File: components/sections/blog/components/Post.tsx [hunk 2]
```diff
@@ -57,12 +57,12 @@ const Post: FC<PostProps> = ({ data, featured }): ReactElement => {
       <div className={`${featured && 'largeTablet:hidden'}`}>
         <GradientBorderWrapper style={{ width: '100%', borderRadius: '5px' }}>
           <div className="w-full relative rounded-[5px] overflow-hidden ">
-            <Image width="580" height={304} src={src as string} alt="Cover" />
+            <Image width="580" height={304} className='min-h-[22rem]' src={src as string} alt="Cover" />
```
Comment (nickytonline) [inline components/sections/blog/components/Post.tsx:60]:
```suggestion
            <Image width={580} height={304} className='min-h-[22rem]' src={src as string} alt="Cover" />
```

**Review by nickytonline — CHANGES_REQUESTED** [review]

File: components/sections/blog/FeaturedPost.tsx (hunk 1 above)
Comment (dev-phantom) [reply components/sections/blog/FeaturedPost.tsx:21]: Okay I didn't add this I met it there I thought it was intentional 

But I'll fix it soon

File: components/sections/blog/components/Post.tsx (hunk 2 above)
Comment (dev-phantom) [reply components/sections/blog/components/Post.tsx:60]: Also this

**Comment by BekahHW** [comment]
> The featured blog post looks good @dev-phantom, I'm just wondering about the actual blog post page. Should we still resize as the page gets bigger or stop at the actual image size? I'm leaning to stop at the actual image size @BekahHW.
> 
> CleanShot 2024-09-10 at 14 33 49 [ CleanShot 2024-09-10 at 14 33 49 ](https://private-user-images.githubusercontent.com/833231/366169772-d0652b2f-cd4b-46d0-ae61-e1d4276502f1.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxNzI4MDcsIm5iZiI6MTcyNjE3MjUwNywicGF0aCI6Ii84MzMyMzEvMzY2MTY5NzcyLWQwNjUyYjJmLWNkNGItNDZkMC1hZTYxLWUxZDQyNzY1MDJmMS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxMlQyMDIxNDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YzIyZTllNzFkMzRlN2YxOTBmMmVlMDkyYmJmMTQzMDA4ZDE2NmYyMGJkMzQ0YmI4NGI1MDY1NDg3NDVlMWZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XcFH-mX0GFrReNkTqcjnlGgO87RD2wMQDc2s2tw3dTs) [ ](https://private-user-images.githubusercontent.com/833231/366169772-d0652b2f-cd4b-46d0-ae61-e1d4276502f1.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxNzI4MDcsIm5iZiI6MTcyNjE3MjUwNywicGF0aCI6Ii84MzMyMzEvMzY2MTY5NzcyLWQwNjUyYjJmLWNkNGItNDZkMC1hZTYxLWUxZDQyNzY1MDJmMS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxMlQyMDIxNDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YzIyZTllNzFkMzRlN2YxOTBmMmVlMDkyYmJmMTQzMDA4ZDE2NmYyMGJkMzQ0YmI4NGI1MDY1NDg3NDVlMWZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XcFH-mX0GFrReNkTqcjnlGgO87RD2wMQDc2s2tw3dTs)

@dev-phantom this is a bit outside of the scope of the original issue, but if you're interested in making this improvement as well, that would be great. Let me know.

**Comment by dev-phantom** [comment]
> > The featured blog post looks good @dev-phantom, I'm just wondering about the actual blog post page. Should we still resize as the page gets bigger or stop at the actual image size? I'm leaning to stop at the actual image size @BekahHW.
> > 
> > CleanShot 2024-09-10 at 14 33 49 [ CleanShot 2024-09-10 at 14 33 49 ](https://private-user-images.githubusercontent.com/833231/366169772-d0652b2f-cd4b-46d0-ae61-e1d4276502f1.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxNzI4MDcsIm5iZiI6MTcyNjE3MjUwNywicGF0aCI6Ii84MzMyMzEvMzY2MTY5NzcyLWQwNjUyYjJmLWNkNGItNDZkMC1hZTYxLWUxZDQyNzY1MDJmMS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxMlQyMDIxNDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YzIyZTllNzFkMzRlN2YxOTBmMmVlMDkyYmJmMTQzMDA4ZDE2NmYyMGJkMzQ0YmI4NGI1MDY1NDg3NDVlMWZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XcFH-mX0GFrReNkTqcjnlGgO87RD2wMQDc2s2tw3dTs) [ ](https://private-user-images.githubusercontent.com/833231/366169772-d0652b2f-cd4b-46d0-ae61-e1d4276502f1.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxNzI4MDcsIm5iZiI6MTcyNjE3MjUwNywicGF0aCI6Ii84MzMyMzEvMzY2MTY5NzcyLWQwNjUyYjJmLWNkNGItNDZkMC1hZTYxLWUxZDQyNzY1MDJmMS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxMlQyMDIxNDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YzIyZTllNzFkMzRlN2YxOTBmMmVlMDkyYmJmMTQzMDA4ZDE2NmYyMGJkMzQ0YmI4NGI1MDY1NDg3NDVlMWZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XcFH-mX0GFrReNkTqcjnlGgO87RD2wMQDc2s2tw3dTs)
> 
> @dev-phantom this is a bit outside of the scope of the original issue, but if you're interested in making this improvement as well, that would be great. Let me know.

"Should we still resize as the page gets bigger or stop at the actual image size? "

What your take on this ma @BekahHW so as to make this improvement

**Comment by dev-phantom** [comment]
please i want to go back to this branch cause i've made updates after this which i added to a different branch 

but if i checkout back to this branch i still get the recent updates i maded in the recent branch there please is there a way i could fix this @nickytonline

**Comment by BekahHW** [comment]
> please i want to go back to this branch cause i've made updates after this which i added to a different branch
> 
> but if i checkout back to this branch i still get the recent updates i maded in the recent branch there please is there a way i could fix this @nickytonline

If you run git log, does it show the commits you don't want?

**Comment by dev-phantom** [comment]
> > please i want to go back to this branch cause i've made updates after this which i added to a different branch
> > but if i checkout back to this branch i still get the recent updates i maded in the recent branch there please is there a way i could fix this @nickytonline
> 
> If you run git log, does it show the commits you don't want?

yes

**Comment by dev-phantom** [comment]
> > please i want to go back to this branch cause i've made updates after this which i added to a different branch
> > but if i checkout back to this branch i still get the recent updates i maded in the recent branch there please is there a way i could fix this @nickytonline
> 
> If you run git log, does it show the commits you don't want?

yes

**Comment by BekahHW** [comment]
@dev-phantom so it looks like maybe you accidentally committed to this branch. You can remove those commits, but make sure you have them in the branch they're supposed to be in so you don't lose your work. [This](https://stackoverflow.com/questions/1338728/how-do-i-delete-a-commit-from-a-branch) might be helpful.

**Comment by dev-phantom** [comment]
> @dev-phantom so it looks like maybe you accidentally committed to this branch. You can remove those commits, but make sure you have them in the branch they're supposed to be in so you don't lose your work. [This](https://stackoverflow.com/questions/1338728/how-do-i-delete-a-commit-from-a-branch) might be helpful.

thank yo so much for this

**Comment by dev-phantom** [comment]
Sorry to ask @nickytonline I was working on the suggestion raised my @BekahHW  and was hoping to add it to this PR 

Do you suggest i just pull the latest update and create a new pr since this wasn't merged or just create a new pr and push the suggestion?

**Comment by nickytonline** [comment]
@dev-phantom , sorry for the confusion. Ignore what I said. I had accidentally created a new PR for some other work with a local copy of your branch. This PR is still good to work on.

**Comment by dev-phantom** [comment]
> @dev-phantom , sorry for the confusion. Ignore what I said. I had accidentally created a new PR for some other work with a local copy of your branch. This PR is still good to work on.

It closed

**Comment by dev-phantom** [comment]
> @dev-phantom , sorry for the confusion. Ignore what I said. I had accidentally created a new PR for some other work with a local copy of your branch. This PR is still good to work on.

It closed

**Pull request closed without merging**